### DISCUSSION TITLE
Sandpack + Team Pro + Artifactory Private Registry + local dev not loading private packages

### DISCUSSION THREAD

**Discussion by rfenderdtn** [codesandbox/sandpack#929] [Q&A]
As the title describes I am using a Sandpack instance with a Team Pro account within a NextJS app trying to load some private packages from my company's private registry on Artifactory within my dev environment running locally on https://localhost:3001 but continually getting this in the preview pane of Sandpack:

```
Couldn't connect to server

This means sandpack cannot connect to the runtime or your network is having some issues. Please check the network tab in your browser and try again. If the problem persists, report it via [email](mailto:hello@codesandbox.io?subject=Sandpack%20Timeout%20Error) or submit an issue on [GitHub.](https://github.com/codesandbox/sandpack/issues)

ENV: create-react-app
ERROR: TIME_OUT
```

Browser devtools shows the request to `design-system-2-6-3-sandpack.codesandbox.io` failing with a 404 and a return payload of `Couldn't find the requested release version @.`

My Sandpack instance in my application looks like this:
```
 <Sandpack
        template="react"
        theme={monokaiPro}
        teamId="design-system"
        customSetup={{
          dependencies: {
            "@mycompany/components": "latest,
          },
        }}
        files={{
          "/App.js": `import {Button} from "@mycompany/components";
  export default function App() {
  return <Button variant="primary">Hello individual</Button>
}`,
        }}
      />
```

In the Codesandbox dashboard I have tried various permutations of the artifactory URL, scopes, even tried adding `localhost` and `localhost:3001` and `https://localhost:3001` to the domains section but nothing is working.

I have confirmed our Artifactory instance is not behind a VPN

![Screen Shot 2023-05-09 at 2 36 50 PM](https://github.com/codesandbox/sandpack/assets/106348204/89d55d83-274a-44a3-a57c-49113a65c016)

**Comment by danilowoz** [comment]
As this looks a bug report and not a discussion, I'm going to address it on https://github.com/codesandbox/sandpack/issues/928
